src/player.js

```diff
@@ -1224,11 +1241,25 @@ export default class Player extends FakeEventTarget {
   _setDefaultTracks(): void {
     const activeTracks = this.getActiveTracks();
     const playbackConfig = this._config.playback;
```

Comment:
don't refer to the config via the private _config since you'll get reference and not copy and it might be dangerous.
`const playbackConfig = this.config.playback;`